runtime/vm/createramclass.cpp

```diff
@@ -2801,24 +2812,21 @@ internalCreateRAMClassFromROMClass(J9VMThread *vmThread, J9ClassLoader *classLoa
 					|| (LOAD_LOCATION_MODULE == locationType))
 				) {
 					U_8* packageUTF = NULL;
-					J9ROMClass *packageRomClass = romClass;
 					U_32 pkgNameLength = 0;
 
-					if (J9ROMCLASS_IS_ARRAY(romClass)) {
-						packageRomClass = elementClass->romClass;
-					}
-
-					packageUTF = J9UTF8_DATA(J9ROMCLASS_CLASSNAME(packageRomClass));
-					pkgNameLength = (U_32) packageNameLength(packageRomClass);
+					pkgNameLength = (U_32) packageNameLength(romClass);
```

Comment:
I had forgotten to remove some code that was used for the array case. I fixed it to make it more clear.